docs/integrations/query.mdx

```diff
@@ -179,7 +179,7 @@ const queryClient = new QueryClient()
 export const App = () => {
   return (
     <QueryClientProvider client={queryClient}>
-      {/* This Provider initalization step is needed so that we reference the same
+      {/* This Provider initialisation step is needed so that we reference the same
```

Comment:
Can it be "initialization"?

Reply:
@dai-shi I thought so, but I decided to change it to be consistent with the rest of the page, where it's written as "initialisation".

Reply:
Oh, I see. Okay, let's leave it for this PR. cc @sandren